### PR TITLE
Fixed code standards issues.

### PR DESCRIPTION
**Pull request by AlexSkrypnyk**
This PR is a continuation of PR #26 and can be merged instead of #26 as it already contains everything from #26.

### CODE REVIEW

**Comment by AlexSkrypnyk** [comment]
The build is failing and there is no way to restart it in Travis (unlike Circle CI where it is possible to do so).

A link to a passing build:
https://travis-ci.org/drupalmel/drupalmel/jobs/94170208

**Comment by Decipher** [comment]
@alexdesignworks You can restart a Travis test quite easily, just hit the retry button against the build:
build__34_-_drupalmel_drupalmel_-_travis_ci

The fails have been logged in an issue, looks related to kattekrab.net, will run the tests locally before merging.

**Comment by AlexSkrypnyk** [comment]
Awesome. Believe it or not, but I could not find this button in TravisCI.

**Comment by Decipher** [comment]
It's possible it's a logged in thing. If you have not already done so, I would suggest logging in and setting up your forks to run tests so that you can ensure your forks pass before creating the PR. Again, something that should probably be documented.

**Comment by AlexSkrypnyk** [comment]
Okay, the reason I did not see the Refresh button is that I'm not a part of the drupalmel/drupalmel girhub team. The button is there for my fork. So, there is still an issue for PR contributors to re-start their builds on Travis.

**Comment by Decipher** [comment]
Not at all, as 99% of the time if the tests fail on your fork they will also fail on the PR and vice versa. Obviously there is an outlier.

However, the plan is that there should be no 'github team' in the future, but rather we should hook into Github's API and expose controls of PR's and tests and such to Slack so anyone can make stuff happen.

File: modules/custom/drupalmel_core/drupalmel_core.install [hunk 1]
```diff
@@ -76,30 +71,24 @@ function drupalmel_core_update_7000() {
 
   // Run this modules persistent update hook.
   drupalmel_core_persistent_update();
+}
 
-  // Add blog feeds.
+/**
+ * Adds blog feeds.
+ */
+function drupalmel_core_update_7001() {
```
Comment (Decipher) [inline modules/custom/drupalmel_core/drupalmel_core.install]: Is there any reason you are changing a hook_update_N() after it has been run? While your change might be more succinct and optimized, it is not how the history of this project went. At one point, the current 7000 was run, and at a later point the current 7001 was run. At no point where either of these new version of the hook_update_N() functions actually run.

Furthermore, while some of the change may be related to coding standards, the re-ordering of the functions is not.

Can you remove this change please?

**Comment by AlexSkrypnyk** [comment]
@Decipher Reverted that update hook changes. Please review.
Could you please comment on code inline if you have any suggestions/questions.

**Comment by Decipher** [comment]
@alexdesignworks Can you possibly remove the DCR stuff as well, as that's in the other PR you've done, and if I where to accept both there would possibly be a conflict. That and it's easier if code changes are only about the issue. Otherwise I can cherry pick it manually. (not inline because the comment applies to multiple files).

**Comment by AlexSkrypnyk** [comment]
If I remove DCR stuff, how do you know that this build passes DCR? Once this one is merged, the other one needs to be closed. This is an exceptional circumstances for such PR since there is an infrastructural and application change.

**Comment by Decipher** [comment]
Happy to accept this PR after you accept https://github.com/alexdesignworks/drupalmel/pull/1 to resolve a minor issue in this PR.

**Pull request merged**